components/article/stylesheets/index.styl

```diff
@@ -445,6 +445,7 @@ body.body-article
       margin-right 20px
     .article-section-callout
       garamond-size('l-body')
+      width "calc(100% - %s)" % (140px)
```

Comment:
Nice use of calc! 👌 

Noticed you are using string interpolation here with a fixed value of `140px` versus just passing the px value in directly. I don't use calc often so I may just be unfamiliar, but would `calc(100% - 140px)` work? Like I said that's really minor and not enough to hold up the PR 😄